v2/commands/job/logs.go

```diff
@@ -0,0 +1,92 @@
+package job
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	oapi "github.com/rescale-labs/htc-cli/v2/api/_oas"
+	"github.com/rescale-labs/htc-cli/v2/common"
+	"github.com/rescale-labs/htc-cli/v2/config"
+	"github.com/rescale-labs/htc-cli/v2/tabler"
+)
+
+func logs(ctx context.Context, c oapi.JobInvoker, projectId, taskId, jobId, pageIndex string) (*oapi.HTCJobLogs, error) {
+	res, err := c.GetLogs(ctx, oapi.GetLogsParams{
+		ProjectId: projectId,
+		TaskId:    taskId,
+		JobId:     jobId,
+		PageSize:  oapi.NewOptInt32(pageSize),
+		PageIndex: oapi.NewOptString(pageIndex),
```

Comment:
You need to not send PageIndex in the request if it's not set. This is the easiest way.

```suggestion
		PageIndex: oapi.OptString{pageIndex, pageIndex != ""},
```